docs/site/Discovering-models.md

```diff
@@ -43,3 +43,20 @@ placed. Default is `src/models`
 
 `--schema`: Specify the schema which the datasource will find the models to
 discover
+
+### Interactive Prompts
+
+After typing `lb4 discover --<your option> [--<arg>]`, based on the option, the
+tool may prompt you for:
+
+- **Name of the connector to discover**: Prompts a list of available
+  connectors(datasources) to choose.
+- **Name of the models to discover**: prompt choices of available models. The
+  answer can be multiple.
+- **Database column naming convention**: By default, LoopBack converts
+  discovered model properties to `camelCase`. You can choose to keep them the
```

Comment:
Suggestion:  
By default, LoopBack converts discovered model properties to `camelCase`. This is recommended. You can choose to...

Reading [Raymond's comment](https://github.com/strongloop/loopback-next/pull/4540/files#r373672012), this might need to change too.